### PR TITLE
Add integration tests for account creation

### PR DESCRIPTION
**Pull request by m-dobler**
My submission for phase 2 of the project. All tests pass 😊 
Big thanks to Tom for the help 🙏

These tests are integration tests that check the http endpoint for creating an account. It tests the endpoint for a completely valid account creation request, as well as possible invalid requests. 

I had to combine multiple invalid inputs to reach the 16 tests. I am not convinced that this provides any value and I think covering the base choice criterion would be sufficient. For this, 7 tests are required and I would be open to deleting the other 9 tests, if requested :+1:

### CODE REVIEW

File: Backend/proptest-regressions/modules/account/tests/create_rt.txt [hunk 1]
```diff
@@ -5,3 +5,4 @@
 # It is recommended to check this file in to source control so that
 # everyone who runs the test benefits from these saved cases.
 cc 890a746ed5cdf24dee6b486359f8b81db34e70d2289187f8339398c1d07fe231 # shrinks to nickname = "A", password = "🂠꜀𖭽 ", mail_prefix = ""
+cc f360da711667485574792904825bcb41e7dd05ccdd20c86a5047713138628957 # shrinks to nickname = "a", password = "𐺰\u{abc}A Ὗ", mail_prefix = ""
```
Comment (Geigerkind) [inline Backend/proptest-regressions/modules/account/tests/create_rt.txt:8]: You used proptest here?

File: Backend/src/modules/account/tests/create_transfer.rs [hunk 2]
```diff
@@ -0,0 +1,617 @@
+use rocket::local::Client;
+use rocket::http::{ContentType, Status};
+use crate::modules::account::dto::{CreateMember, Credentials};
+use crate::modules::account::Account;
+use mysql_connection::tools::{Exists, Execute};
+
+#[test]
+fn create_account_nick_valid_email_valid_password_valid() {
+    // Given
+    let account = Account::default();
```
Comment (Geigerkind) [inline Backend/src/modules/account/tests/create_transfer.rs:10]: There is a lot of duplicated code. You could also use private functions to shorten it. I will look into a JUnit like syntax crate though soon.

File: Backend/proptest-regressions/modules/account/tests/create_rt.txt (hunk 1 above)
Comment (m-dobler) [reply Backend/proptest-regressions/modules/account/tests/create_rt.txt:8]: No, I didn't use proptest for my tests, I only executed all tests on the `dev` branch and the `create_rt` proptest test failed. Seems like the cause are passwords with weird characters?

File: Backend/src/modules/account/tests/create_transfer.rs (hunk 2 above)
Comment (m-dobler) [reply Backend/src/modules/account/tests/create_transfer.rs:10]: > You could also use private functions to shorten it.
Will do (but not before the weekend 🤯 )

**Pull request merged**